src/applications/caregivers/utils/selectors/feature-toggles.js

```diff
@@ -0,0 +1,17 @@
+import { createSelector } from 'reselect';
+import { toggleValues } from '@department-of-veterans-affairs/platform-site-wide/selectors';
+import FEATURE_FLAG_NAMES from '@department-of-veterans-affairs/platform-utilities/featureFlagNames';
+
+const selectFeatureToggles = createSelector(
+  state => ({
+    isLoadingFeatureFlags: state?.featureToggles?.loading,
+    isBrowserMonitoringEnabled: toggleValues(state)[
+      FEATURE_FLAG_NAMES.hcaBrowserMonitoringEnabled
```

Comment:
Lets go ahead and decouple this from the EZ & EZR by creating its own feature flag.

Reply:
https://github.com/department-of-veterans-affairs/vets-api/pull/15459